docs/community/com-query/grafana-connector.md

```diff
@@ -112,45 +119,15 @@ If your environment requires a different installation method, please refer to th
 <!-- TODO: dataset-specific examples for each "format as" option -->
 <!-- TODO: screenshots of config and results -->
 
-
 ### Querying
```

Comment:
This is another totally different task which should be in its own page

Reply:
@rachithrr you can skip this